Assets/Scripts/Map/Generators/Patrolling/Partitioning/MeetingPoints/MeetingPoint.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+
+namespace Maes.Map.Generators.Patrolling.Partitioning.MeetingPoints
+{
+    public class MeetingPoint
+    {
+        public MeetingPoint(int vertexId, int atTicks)
+        {
+            VertexId = vertexId;
+            AtTicks = atTicks;
+        }
+
+        public int VertexId { get; }
+        public int AtTicks { get; set; }
+        public HashSet<int> RobotIds { get; } = new();
```

Comment:
RobotIds should be immutable. If it is possible take a hashset in in the constructor and make this an IReadOnlyCollection or IReadOnlyList. Otherwise we might accidentally smuggle information between robots.

Reply:
Now i see it AtTicks should also be have its setter removed.